lit_gpt/config.py

```diff
@@ -1167,7 +1167,27 @@ def norm_class(self) -> Type:
         _mlp_class="LLaMAMLP",
         intermediate_size=5632,
         n_query_groups=4,
-    )
+    ),
+    # chat version has a slightly larger vocab size
```

Comment:
How do you know you need to set it on `padded_vocab_size` and not just `vocab_size=32003`?

Reply:
I suggest moving the comment down to the line where it applies

Reply:
> How do you know you need to set it on padded_vocab_size and not just vocab_size=32003?

Glad that you've asked.

If to set only `vocab_size=32003` then it will be automatically padded to `32064`.

But here one more thing: I specified two variables, not just one.
https://github.com/Lightning-AI/lit-gpt/blob/0faa93435a91bd72b62248ac290caf2b37719774/lit_gpt/config.py#L1176-L1177

That's because the model used a tokenizer with size of 32k and the fine-tuned model extended the size by 3 tokens: https://huggingface.co/TinyLlama/TinyLlama-1.1B-Chat-v0.5/blob/main/added_tokens.json. That means that the `huggingface.AutoTokenizer.from_pretrained` will have a size of 32k and the model 32003.

In Lit-GPT in models only `padded_vocab_size` is used.
The only place where `vocab_size` is used is in the tests/test_tokenizer.py
https://github.com/Lightning-AI/lit-gpt/blob/2647dbda9e29c8a6e1d7a5f38f181897a893a8dd/tests/test_tokenizer.py#L51-L52

So in order the model and tests to work I specified both variables.

Reply:
> I suggest moving the comment down to the line where it applies

I decided to do so that it's clearer why we have a separate config instead of having a template and then formatting it in the loop.

Initially I did it with the template: https://github.com/Lightning-AI/lit-gpt/pull/778/commits/d5e1ff43da96e770eb5372a834b65eeefb7170b0

@awaelchli How do you think: if I extend this "loop approach" with additional logic 
```python
if kind == "chat":
    copy["padded_vocab_size"] == 32003
```
wouldn't it be too "ugly"?